### ISSUE TITLE
Primitives for Random Oversampling or Undersampling of Data

### ISSUE THREAD

**Issue by sagarshankar** [datamllab/tods#77]
Do any of the TODS primitives or any of the primitives within the d3m infrastructure perform Random Sampling like Undersampling or Oversampling? I have been trying to incorporate those steps within a pipeline prior to sending the data into the detection algorithm?

**Comment by lhenry15** [comment]
Nope. At this point, we are also figuring out how to do that and see if another module is needed. Because oversampling for time series data is very different from oversampling for tabular/image data. There is a survey paper working on this. But how to incorporate that into our setting need some more investigations. Feel free to throw some ideas here. https://arxiv.org/pdf/2002.12478.pdf
